src/storage/mvcc/txn.rs

```diff
@@ -1109,6 +1125,25 @@ mod tests {
         test_mvcc_txn_commit_err_imp(b"k2", &long_value);
     }
 
+    #[test]
+    fn test_min_commit_ts() {
+        let engine = TestEngineBuilder::new().build().unwrap();
+
+        let (k, v) = (b"k", b"v");
+        let ts = super::super::compose_ts;
+
+        must_prewrite_put_for_large_txn(&engine, k, v, k, ts(10, 0), 100, 0);
+        must_check_txn_status(&engine, k, ts(10, 0), ts(20, 0), ts(20, 0), 100, 0);
+        // The the min_commit_ts should be ts(20, 1)
+        must_commit_err(&engine, k, ts(10, 0), ts(15, 0));
+        must_commit_err(&engine, k, ts(10, 0), ts(20, 0));
+        must_commit(&engine, k, ts(10, 0), ts(20, 1));
+
+        must_prewrite_put_for_large_txn(&engine, k, v, k, ts(30, 0), 100, 0);
+        must_check_txn_status(&engine, k, ts(30, 0), ts(40, 0), ts(40, 0), 100, 0);
+        must_commit(&engine, k, ts(30, 0), ts(50, 0));
```

Comment:
What are these lines for?

Reply:
I'm trying to cover `commit_ts < min_commit_ts`, `commit_ts == min_commit_ts` and  `commit_ts > min_commit_ts`, and this is the last case.

Reply:
Got it.